show/main.py

```diff
@@ -1356,16 +1356,29 @@ def show_run_snmp(db, ctx):
 @runningconfiguration.command()
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
 def syslog(verbose):
-    """Show Syslog running configuration"""
+    """Show Syslog running configuration
+    To match below cases:
+    *.* @IPv4:port
+    *.* @[IPv4]:port
+    *.* @[IPv6]:port
+    """
     syslog_servers = []
     syslog_dict = {}
+    re_ipv4_1 = re.compile(r'^\*\.\* @(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}):\d+')
+    re_ipv4_2 = re.compile(r'^\*\.\* @\[(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\]:\d+')
+    re_ipv6 = re.compile(r'^\*\.\* @\[([0-9a-fA-F:]+)\]:\d+')
     with open("/etc/rsyslog.conf") as syslog_file:
         data = syslog_file.readlines()
     for line in data:
-        if line.startswith("*.* @"):
-            line = line.split(":")
-            server = line[0][5:]
-            syslog_servers.append(server)
+        if re_ipv4_1.match(line):
+            server =  re_ipv4_1.match(line).group(1)
+        elif re_ipv4_2.match(line):
+            server =  re_ipv4_2.match(line).group(1)
+        elif re_ipv6.match(line):
+            server =  re_ipv6.match(line).group(1)
+        else:
+            continue
```

Comment:
>continue [](http://example.com/codeflow?start=12&length=8)

Should we report error message on stderr instead of being silent?

Reply:
I don't think so. I am checking all lines in rsyslogd.conf.
Even if only check lines start with `*.* @`, it is possible we have tcp format server starts with `*.* @@`. It is not enabled in the rsyslogd.conf.
Seems current `show run syslog` command only care about UDP server